Source/BSGOutOfMemoryWatchdog.m

```diff
@@ -114,6 +122,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
     self.cachedFileInfo[@"app"][@"releaseStage"] = change[NSKeyValueChangeNewKey];
     [self writeSentinelFile];
 }
+
+- (void)handleTransitionToActive:(NSNotification *)note {
+    self.cachedFileInfo[@"app"][@"isActive"] = @YES;
+    [self writeSentinelFile];
```

Comment:
Can this IO be moved to a background thread?

Reply:
It’s already on a background thread to actually write the file, the thread is managed within the file writing function.